spatial/r1/bound.go

```diff
@@ -2,9 +2,9 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package bound
+package r1
 
-// Bound represents [Min, Max] bounds.
-type Bound struct {
+// Interval represents [Min, Max] bounds.
```

Comment:
@vladimir-ch Do we want to be this definitive? We use them as [min, max] bounds, but they could be any type of real interval.

Reply:
I would prefer leaving it unspecified, it was the main motivation behind these changes. But what will we put there instead? `Interval is an interval`? :-)

Reply:
Simple, direct and to the point. SGTM ("Interval is an interval in R1" seems like over cooking.)